src/modules/components/proposal/proposalActions/proposalActionsTypes/proposalActionTokenWithdraw.ts

```diff
@@ -0,0 +1,51 @@
+import type { ICompositeAddress } from '../../../../types';
```

Comment:
We renamed this file to `proposalActionWithdrawToken`, this is defining the same interface with a different file name